xsd/siri_model/siri_situationActions.xsd

```diff
@@ -203,17 +203,22 @@ Rail transport, Roads and road transport
 			<xsd:documentation>Type for list of SITUATIONs.</xsd:documentation>
 		</xsd:annotation>
 		<xsd:sequence>
-			<xsd:element name="Name" type="xsd:NMTOKEN">
+			<xsd:element name="Name" type="xsd:NMTOKEN" minOccurs="0">
 				<xsd:annotation>
 					<xsd:documentation>Name of action data Element.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="ActionCode" type="xsd:NMTOKEN" minOccurs="0">
```

Comment:
To me it still looks like we will end up with ActionCode in PassengerInformationAction that already has ActionRef. 

I propose that we separate the ParameterisedActionStructure (and also the ActionDataStructure) in one structure (without the ActionCode) that will be the base of the PassengerInformationActionStructure and another one (with the ActionCode) that will be the base of the PushedActionStructure and all the related "PushTo..." structures. I don't think there's a simpler solution since extension and restriction are mutually exclusive. An alternative would be to replace the extension with groups.